.github/pull_request_template.md

```diff
@@ -1,7 +1,17 @@
 <!--- Provide a general summary of your changes in the Title above -->
 
 ## Description
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail. If it's a Big PR, remove this section -->
+
+## Description (Big PR)
+<!--- Mark what's included in this PR and describe it. If it isn't a Big PR, remove this section -->
+* [ ] Multiple features (which ones? how were they tested/how to test?).
+* [ ] Change of the scope (why?).
+* [ ] Introduces breaking changes (which ones?).
+* [ ] Multiple files with code changes.
+* [ ] Multiple files just moved around.
+
+<!--- Detail all the points marked above. Refer to "Big PRs" in https://csmdigital.atlassian.net/wiki/spaces/CSMP/pages/2490695681/New+Team+Agreements#Team-Technical-Agreements.1 for more info. -->
```

Comment:
I would put this as a link, rather than as a comment

Reply:
Do you know if links in comments actually show? Because I tried to preview it in VS Code and, because it's a comment, it wasn't showing to me how it would look. The preview doesn't show comments, and the raw makes it pointless to make it a markdown link. So I left it as a raw link just to be safe.

(I also preferred a link, it's just that I couldn't see the final result)

Reply:
Markdown is pretty much parsed as HTML in GitHub. `<!-- -->` indicates comments so it will never show up as a link. If you'd prefer having this emphasised, I suggest to put such "comments" as [quote highlight](https://github.com/orgs/community/discussions/16925) such as:

```md
> [!NOTE]
> Detail all the points marked above. Refer to "Big PRs" in [Team Agreements](https://csmdigital.atlassian.net/wiki/spaces/CSMP/pages/2490695681/New+Team+Agreements#Team-Technical-Agreements.1) for more info
```

which would results similar to:

> [!NOTE]
> Detail all the points marked above. Refer to "Big PRs" in [Team Agreements](https://csmdigital.atlassian.net/wiki/spaces/CSMP/pages/2490695681/New+Team+Agreements#Team-Technical-Agreements.1) for more info

Reply:
@robertoHeCi @matyasjay My idea was to have a comment like we have in other sections. Something that the creator of the PR can see as a guide but that, when put to review, the rest of the people doesn't see (without having to delete the line). Just so that the info is there but without adding noise to the PR description. If comments don't accept links, I'd rather leave the link as a raw text.